recipe/setupext.patch

```diff
@@ -1,16 +1,15 @@
-diff --git setupext.py setupext.py
-index 0ae553a..66c6e02 100755
---- setupext.py
-+++ setupext.py
-@@ -1425,6 +1425,7 @@ class BackendTkAgg(OptionalBackendPackage):
-             tk.destroy()
- 
-         # Save directories and version string to cache
-+#       tcl_lib_dir = tk_lib_dir = r'C:\Tcl%d' % (8 * tuple.__itemsize__)
-         self.tcl_tk_cache = tcl_lib_dir, tk_lib_dir, str(Tkinter.TkVersion)[:3]
-         return self.tcl_tk_cache
- 
-@@ -1531,13 +1532,10 @@ class BackendTkAgg(OptionalBackendPackage):
+--- setupext.py	2016-01-10 19:20:20.000000000 -0300
++++ setupext.py	2016-05-14 16:29:26.477374144 -0300
+@@ -990,7 +990,7 @@
+             ]
+         ext = make_extension('matplotlib._png', sources)
+         pkg_config.setup_extension(
+-            ext, 'libpng', default_libraries=['png', 'z'],
++            ext, 'libpng', default_libraries=['libpng', 'z'],
```

Comment:
Agree with @ocefpaf. The point of conda-forge is that we have CI running, and we can actually see the problems that arise for at least one usecase. It is as much about collective learning as it is providing a useable package source.

I think we can merge this as it is @ocefpaf (it is working). The question we may want to subsequently answer though is - do we need to make this change in matplotlib also?